parm/config/gefs/config.base

```diff
@@ -38,7 +38,8 @@ export FIXugwd=${FIXgfs}/ugwd
 export PACKAGEROOT="@PACKAGEROOT@"    # TODO: set via prod_envir in Ops
 export COMROOT="@COMROOT@"    # TODO: set via prod_envir in Ops
 export COMINsyn="@COMINsyn@"
-export BASE_CPLIC="@BASE_CPLIC@"
+export ICSDIR="@ICSDIR@"
```

Comment:
This is being set in `config.stage_ic`. Why is it here, and templated to boot?

Reply:
I have now moved both `BASE_IC` and `ICSDIR` from `config.base` to `config.stage_ic`. Updated `workflow/setup_expt.py` to accommodate the variable move. The `ICSDIR` is either provided by the user via `--icsdir` or is the default `''`. If `ICSDIR` is not set by the user then it is defined in the remaining part of `config.stage_ic`.

Reply:
Committed to my branch @ a0b9af07